### PR TITLE
[home-assistant] Bump HAAS to 0.114.0

### PR DESCRIPTION
**Pull request by damacus**
#### Checklist

- [x] Chart Version bumped
- [x] Title of the PR starts with chart name

### CODE REVIEW

File: charts/home-assistant/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: 0.113.3
+appVersion: 0.114.0
 description: Home Assistant
 name: home-assistant
-version: 1.1.0
+version: 1.1.1
```
Comment (Juggels) [inline charts/home-assistant/Chart.yaml:5]: Shouldn't this bump to 1.2.0 as this isn't really a bugfix but actually introduces a change to the chart?

Comment (damacus) [reply charts/home-assistant/Chart.yaml:5]: I'm happy either way to be honest. I've been bumping the HAAS version and incrementing the chart as a patch version.

**Comment by carpenike** [comment]
@damacus -- do you want me to incorporate this into my PR and we can close this one?

File: charts/home-assistant/Chart.yaml (hunk 1 above)
Comment (billimek) [reply charts/home-assistant/Chart.yaml:5]: Personally, I've been bumping only the patch version as well whenever updating any chart's image version.

I suggest that we reserve minor semver changes (e.g. going from 1.1.0 to 1.2.0) for functional changes to the _chart_ itself and not the underlying application changes.

**Pull request merged**